### ISSUE TITLE
Serialization of `TorchForce` is not self-contained

### ISSUE THREAD

**Issue by raimis** [openmm/openmm-torch#65]
When serializing `TorchForce`, the file name of the PyTorch model is save rather its content:
```python
from openmmtorch import TorchForce
from openmm import XmlSerializer

force = TorchForce('filename.pt')
print(XmlSerializer.serialize(force))
```
```xml
<?xml version="1.0" ?>
<Force file="filename.pt" forceGroup="0" outputsForces="0" type="TorchForce" usesPeriodic="0" version="1">
	<GlobalParameters/>
</Force>
```

So, it is a user responsibility to ensure that the right model file is in the right place during desalinization.

Ideally, the serialized object should be self-contained. We should save the file content, not it's name. Or, at least, we should store a hash of the file content and verify it during desalinization.

**Comment by jchodera** [comment]
The [TorchScript format](https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/docs/serialization.md) is actually a ZIP file, so it is a binary format. We can simply `base64` encode/decode this file. It's [simple to implement in C++11](https://gist.github.com/williamdes/308b95ac9ef1ee89ae0143529c361d37), and there is a [proposed `std::string base64_encode/decode` addition](https://renenyffenegger.ch/notes/development/Base64/Encoding-and-decoding-base-64-with-cpp/index) that would further simplify this in future.

**Comment by raimis** [comment]
@RaulPPelaez I guess, this was solved by #97 too.
